### PR TITLE
Add capabilities support to bitbar utils

### PR DESCRIPTION
**Pull request by Cawllec**
## Goal

Adds support for defining and adding regular and device capabilities, as well as a couple of changes to the bitbar tunnel logic to ensure it connects correctly.

### CODE REVIEW

File: lib/maze/bitbar_devices.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Maze
+  # Provides a source of capabilities used to run tests against specific BrowserStack devices
+  # noinspection RubyStringKeysInHashInspection
+  class BitBarDevices
+    APPIUM_1_9_1 = '1.9.1'
+    APPIUM_1_15_0 = '1.15.0'
+    APPIUM_1_20_2 = '1.20.2'
```
Comment (twometresteve) [inline lib/maze/bitbar_devices.rb:9]: Looks like these are unused?

File: lib/maze/bitbar_devices.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Maze
+  # Provides a source of capabilities used to run tests against specific BrowserStack devices
```
Comment (twometresteve) [inline lib/maze/bitbar_devices.rb:4]:
```suggestion
  # Provides a source of capabilities used to run tests against specific BitBar devices
```

**Review by twometresteve — APPROVED** [review]
LGTM, just a couple of tiny comments.

File: lib/maze/bitbar_devices.rb (hunk 1 above)
Comment (Cawllec) [reply lib/maze/bitbar_devices.rb:9]: This is configurable for Android, so will probably come up with the next step of running Android apps

**Pull request merged**